### DISCUSSION TITLE
Anyone Have aRS485 Daisy Chain Wiring Example they can share?

### DISCUSSION THREAD

**Discussion by sjames99** [aqualinkd/AqualinkD#301] [Q&A] [answered]
I understand the concept of daisy chaining the devices, but have never done it when you have more devices than terminal blocks.  So in this case 5 devices and 2 blocks.  I am not a wiring expert, border on an intermediate one 😄.  I was hoping someone had a photo of their aqualink wiring with a daisy chain as the diagrams are just clouding my mind and keep second guessing myself.  I did try it once and lost all connection to all devices so quickly reverted it last year.

**Comment by sfeakes** [comment, marked as answer]
I don't have any pic's.  But it's as simple as putting both sets of wires into the device terminal block.  So would look like.
The picture in the wiki is also a good depiction. https://github.com/sfeakes/AqualinkD/wiki#RS485-adapters
```
Panel Terminal Block -------- Device #1 Terminal Block -------- Device #2 Terminal Block.
```
(Device \# 1 simply has 2 sets of wires in it's terminal block.)
As opposed to below where the panel terminal block has both sets of wires.

```
Panel Terminal Block 
                   |-------- Device #1 Terminal Block 
                   |-------- Device #2 Terminal Block
```


To use daisy chain, you really need to know which device is terminated though, as only the first and last can be terminated.
Also remember that the 2 panel terminal blocks (if you have 2) are hard wired together, so treat them as 1 (just makes it easier to connect devices.
You can also mix this up if needed if you know what is and isn't terminated.
Below is how my setup looks.
```
Panel Terminal Block #1
                   |-------- Pentair VSP 
                   |-------- Jandy LXi heater
Panel Terminal Block #2
                   |-------- Haywood SWG 
                   |-------- AqualinkD USB --------- AqualinkD2 USB --------- Jandy Allbutton Keypad
```
Both the AqualinkD USB's are not terminated, everything else is. (I only have 2 for debugging purposes)

**Reply by sjames99** [reply]
Thank you!

I have to give it another try, I have six sets of wires coming in to two terminal blocks.  I haven't been able to add any additional devices to the first terminal block.  Nothing will work if I add another device.  So that leaves with four devices on the last terminal block.
